### DISCUSSION TITLE
Why is `GeoM.Rotate` clockwise?

### DISCUSSION THREAD

**Discussion by musjj** [hajimehoshi/ebiten#2792] [Q&A] [answered]
I thought the positive direction of rotation is conventionally counter-clockwise. But it seems that `Geom.Rotate` rotates in the clockwise direction, so I found it surprising. Is this some kind of convention in game programming?

**Comment by hajimehoshi** [comment, marked as answer]
Yes, in Ebitengine, this is clockwise i.e. the Y axis is downward. This is opposite to math, but is usual in game programming. See also https://ebitengine.org/en/documents/matrix.html
